src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -94,29 +98,30 @@ public void TestInit()
         [TestMethod]
         public void ErrorListInfoBarController_Ctor_NullHost_Throws()
         {
-            // Arrange
-            Action act = () => new ErrorListInfoBarController(null, this.unboundProjectFinder);
-
-            // Act & Assert
+            Action act = () => new ErrorListInfoBarController(null, this.unboundProjectFinder, this.logger);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("host");
         }
 
         [TestMethod]
         public void ErrorListInfoBarController_Ctor_NullBindingProvider_Throws()
         {
-            // Arrange
-            Action act = () => new ErrorListInfoBarController(this.host, null);
-
-            // Act & Assert
+            Action act = () => new ErrorListInfoBarController(this.host, null, this.logger);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("unboundProjectFinder");
         }
 
+        [TestMethod]
+        public void ErrorListInfoBarController_Ctor_NullLogger_Throws()
+        {
+            Action act = () => new ErrorListInfoBarController(this.host, this.unboundProjectFinder, null);
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("logger");
+        }
+
         [TestMethod]
         public void ErrorListInfoBarController_Refresh_ActiveSolutionBoundAndFullyLoaded_HasNoUnboundProjects()
         {
             // Arrange
             this.SetBindingMode(SonarLintMode.LegacyConnected);
-            var testSubject = new ErrorListInfoBarController(this.host, this.unboundProjectFinder);
```

Comment:
The controller now needs the logger as a constructor parameter.
I changed all of the existing tests to call a helper method to create the test subject rather than add the new parameter inline.